packages/lit-dev-content/site/docs/ssr/overview.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Server-side rendering (SSR)
+eleventyNavigation:
+  key: 🧪 Overview
+  parent: Server-side rendering
+  order: 1
+---
+
+<details class="pre-release">
+  <summary> 🧪 Pre-release software: not for production.</summary>
+  
+  Lit labs packages are published to get feedback from the wider community. This code shouldn't be used in production, but we encourage you to try it out and [file issues](https://github.com/lit/lit/issues/new/choose) for any bugs you find. For general feedback, please use the GitHub [discussion](https://github.com/lit/lit/discussions).
+
+  For more information about the Lit labs process, see [Lib Labs](/docs/libraries/labs/)
+
+</details>
+
+Server-side rendering (SSR) can help make pages work without JavaScript, either for robustness or performance reasons, or to work with web crawlers and bots that don't run a full browser. Some sites can render faster if they render static markup first, and optionally hydrate the components afterwards by loading the JavaScript.
+
+[Lit SSR](https://github.com/lit/lit/tree/main/packages/labs/ssr#readme) lets you render Lit components and templates to static HTML markup in non-browser JavaScript environments like Node. It works without fully emulating the browser's DOM, and takes advantage of Lit's declarative template format to support streaming.
+
+Server-side rendering Lit elements also utilizes [Declarative Shadow Roots](https://web.dev/declarative-shadow-dom/) which is a new browser feature that is [shipping in Chrome](https://developer.chrome.com/blog/new-in-chrome-90/#declarative) which allows browsers to parse and attach shadow roots just by parsing HTML without the need for JavaScript. A polyfill is available for browsers without support yet whose usage is detailed in the [client API](/docs/ssr/client-api#lit-components) documentation.
```

Comment:
```suggestion
Server-side rendering Lit elements utilizes [Declarative Shadow DOM](https://web.dev/declarative-shadow-dom/), a new HTML feature that is [shipping in Chrome and Edge](https://developer.chrome.com/blog/new-in-chrome-90/#declarative) which allows HTML to attach shadow roots to element without the need for JavaScript.

Until all browsers ship declarative shadow DOM, a very small polyfill is available that can be inlined into your page. This lets you use SSR for robustness now and incrementally address non-JS use cases as the feature is rolled out in other browsers and crawlers. Usage of the polyfill is detailed in the [client API](/docs/ssr/client-api#lit-components) documentation.
```

Reply:
"to element" => "to elements"
"non-JS" => "non-JavaScript"

Reply:
The middle sentence is still a little confusing.  Basically, we're saying that using it with the polyfill gives you:

- The benefits of robustness _on browsers that support declarative shadow DOM_, BUT
- It will continue to work on browsers that don't support it, as long as they can run JavaScript, AND
- As browsers implement declarative shadow DOM, they'll be able to take advantage of it without you changing your code.

That is an awful lot to put in one sentence. I think starting with "robustness now" makes it confusing. The previous list equates "Robustness" with "works without JavaScript," but that only works on browsers that have native declarative shadow DOM--exactly where you don't need the polyfill. And the polyfill, naturally, doesn't work if you're not running JS. It seems to me like "robustness" is the least relevant benefit here.

Maybe instead something like:

This lets you use SSR now, across a variety of browsers:

- On browsers and crawlers that support declarative shadow DOM, the components should display correctly with or without JavaScript enabled.

- On browsers and crawlers that don't support declarative shadow DOM, the components will display correctly as long as JavaScript is enabled.

??

Reply:
I get what you're saying but I think the first bullet point is unnecessary since we've already explained DSD is a browser feature that doesn't require JS. This sentence just needs to assure users that a polyfill is available for browsers that don't support it, albeit with a small inlined JS. Maybe we can drop the "robustness" and reword to highlight that SSR can be used for any browsers with JS right now.

> Until all browsers include declarative shadow DOM support, a very small polyfill is available that can be inlined into your page. This lets you use SSR now for any browsers and crawlers with JavaScript enabled and incrementally address non-JavaScript use cases as the feature is rolled out to the rest. Usage of the polyfill is detailed in the [client API](/docs/ssr/client-api#lit-components) documentation.

Reply:
"robustness" is the page not requiring external resources that could fail to load. The polyfill helps with robustness because we recommend that it's inlined.